your-rents-services-geodata/src/test/java/com/yourrents/services/geodata/repository/CountryRepositoryTest.java

```diff
@@ -0,0 +1,37 @@
+package com.yourrents.services.geodata.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.yourrents.services.geodata.TestYourRentsGeoDataServiceApplication;
+import com.yourrents.services.geodata.model.Country;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Import;
+import org.springframework.test.context.TestPropertySource;
+
+@SpringBootTest
+@Import(TestYourRentsGeoDataServiceApplication.class)
+@TestPropertySource(properties = "spring.config.additional-location=classpath:application-test.properties")
```

Comment:
Do you mind to use `@ActiveProfiles("test")` instead?

To me It seems a little verbose and confusing: adding a property for adding a file of properties.